### PR TITLE
[MIG] 12.0 base country state translatable

### PR DESCRIPTION
**Pull request by ddufresne**

### CODE REVIEW

**Comment by ddufresne** [comment]
@pedrobaeza where do I sign the CLA ?

**Comment by pedrobaeza** [comment]
Check the indicated link for details

**Comment by nikul-serpentcs** [comment]
> @pedrobaeza where do I sign the CLA ?

https://odoo-community.org/page/cla

File: base_country_state_translatable/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# © initOS GmbH 2017
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Translate Country States",
+    "version": "12.0.1.0.0",
+    "depends": ['base',
```
Comment (nikul-serpentcs) [inline base_country_state_translatable/__manifest__.py]:
```suggestion
"depends": ['base',]
"website": 'https://github.com/OCA/partner-contact',
"summary": "Translate Country States",
```

File: base_country_state_translatable/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# © initOS GmbH 2017
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Translate Country States",
+    "version": "12.0.1.0.0",
+"depends": ['base',],
```
Comment (tarteo) [inline base_country_state_translatable/__manifest__.py]: Can you indent here, please?

File: base_country_state_translatable/models/res_country.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# © initOS GmbH 2017
```
Comment (tarteo) [inline base_country_state_translatable/models/res_country.py]: Can you replace the copyright sign with 'Copyright'. And also in the other files.

**Review by tarteo — CHANGES_REQUESTED** [review]
Thanks. Just some small things otherwise :+1:

File: base_country_state_translatable/__manifest__.py (hunk 2 above)
Comment (ddufresne) [reply base_country_state_translatable/__manifest__.py]: @tarteo fixed

File: base_country_state_translatable/models/res_country.py (hunk 3 above)
Comment (ddufresne) [reply base_country_state_translatable/models/res_country.py]: @tarteo fixed

**Review by yostashiro — APPROVED** [review]
Looks good to me.

**Review by tarteo — APPROVED** [review]
Thanks!

**Pull request merged**